chainer/iterators/random_state.py

```diff
@@ -0,0 +1,89 @@
+import contextlib
+import threading
+
+import numpy
+
+
+_random_state = threading.local()
+_random_state.value = None
+
+
+def get_random_state():
+    """Get a :class:`~numpy.random.RandomState` during iteration.
+
+    The instance obtained is unique to each index on a batch.  You can
```

Comment:
From the same reason as the previous one, I would suggest to describe it as one of expected use cases.